client/src/pages/Homepage/useStyles.ts

```diff
@@ -0,0 +1,67 @@
+import { Grid, makeStyles } from '@material-ui/core';
+import homepage_image from '../../Images/homepage_image_v2.png';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    minHeight: '100vh',
+  },
+  wrapper: {
+    minHeight: '100vh',
+    minWidth: '100vw',
+    [theme.breakpoints.down('xs')]: {
+      paddingTop: '50px',
+    },
+  },
+  gridContainer: {
+    height: '100vh',
+  },
+  contentWrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+  },
+
+  form: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: '15px',
+    maxWidth: '500px',
+    margin: 'auto',
+    '& h1': {
```

Comment:
Optional: This and some of the styles can be set within MUI's theme to ensure consistent font sizes across the app :)

Reply:
Yes, that makes sense, thank you! - I was concerned about accidentally changing other components that I am unaware of.  As we move forward I will consider that more carefully, as it does seem to make the most sense in not duplicating code, etc...